mesheryctl/cmd/version.go

```diff
@@ -35,9 +35,8 @@ var versionCmd = &cobra.Command{
 	Short: "Version of mesheryctl",
 	Long:  `Version of Meshery command line client - mesheryctl.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		log.Info("Client Version: ", BuildClient)
+		log.Info("Client Version: %v \t  GitSHA: %v", BuildClient, CommitSHA)
```

Comment:
`log.info` should be replaced with `log.infof`
For safe practice, please always try it on local by creating a binary using `make` command.

Reply:
Did the change..